convert_hcl_to_json.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+install_jq(){
+    OS=$(uname -s)
+    if [[ "$OS" == "Darwin" ]]; then
+        OS="macos"
+    elif [[ "$OS" == "Linux" ]]; then
+        OS="linux"
+    else
+        echo "Unsupported OS: $OS"
+        exit 1
+    fi
+
+    ARCH=$(uname -m)
```

Comment:
This will create issue on arch that is `x86_64`, `aarch64`, `armv7l`, and potentially on `i686`. You could create function that detects arch and substitutes these values if necessary.

```bash
_detect_arch() {
  local machine="$(uname -m)"

  declare -A arch_map=(
    ["x86_64"]="amd64"
    ["aarch64"]="arm64"
    ["armv7l"]="arm"
    ["i686"]="i386"
  )

  # Get arch from map if defined, otherwise don't replace
  # For example, in case of arm64, it just echo-es arm64
  echo "${arch_map[$machine]:-$machine}"
}
``` 
and then call it to get proper arch
``` bash
ARCH="$(_detect_arch)"
```